content/project-news/reactos-participating-google-summer-code-2019.html

```diff
@@ -4,7 +4,7 @@
 date:        2019-02-28
 aliases:     [ "/node/69610" ]
 tags:        [ "gsoc", "gsoc-2019" ]
-banner:      "img/project-news/reactos-participating-google-summer-code-2019-thumb.png"
+banner:      "img/project-news/google-summer-code-thumb.png"
```

Comment:
why?

Reply:
To avoid duplicating the same thumbnail picture. Also the picture itself doesn't have anything connected with 2019 year.